packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/utils/distant-tables-sorter.util.ts

```diff
@@ -0,0 +1,19 @@
+import { RemoteTableStatus } from 'src/engine/metadata-modules/remote-server/remote-table/dtos/remote-table.dto';
+
+export const distantTablesSorter = (table1, table2) => {
```

Comment:
sortRemoteTable?

utils naming should be actions
(is distant a new business terminology I don't now about)

Reply:
Yes we have :
- distant tables which are the tables in the other database
- foreign tables
- remote tables which are distant tables synced. They makes the link between a foreign and a distant table

Reply:
ok!

Reply:
Actually, I am not 100% convince we need this distant table name.
I feel it is just remoteTable and what we have in db is remoteTableMetadata

to me it's exactly the same thing as object vs ObjectMetadata. the remoteTable does already represent the table in the other database. it has extra info but is the same object. Then in your code, you want to distinguish the remoteTable entry in our db (aka remoteTableMetadata) from the real remote Table

Reply:
So you prefer:
- remote tables for tables of the other DB
- remoteTableMetdata for the join table we store?
I feel like `remoteTableMetdata` would be a bit confusing. This is probably this join that was wrongly named in the first place

Reply:
we can merge this PR without changing this as it's not introduced here

but I feel remoteTableMetadata is the same as remoteTable (it doesn't matter it contains a bit more sync information). but as ObjectMetadata is describing an Object (with properties, it's kind of a projection of the object concept in our code: defaultValue + type + settings, etc. without really capturing the essence of the object itself), the RemoteTableMetadata could be a set of properties describing the real remoteTable.
Let's discuss it tomorrow, I'm not 100% convince by remoteTableMetadata